### PR TITLE
[TxPool] Return capacity in InspectResponse

### PR DESCRIPTION
**Pull request by dbrajovic**
# Description

Return the status of pool capacity when inspecting the TxPool with `txpool_inspect`.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [ ] I have tested this code
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: jsonrpc/blockchain.go [hunk 1]
```diff
@@ -58,6 +58,9 @@ type blockchainInterface interface {
 	// GetNonce returns the next nonce for this address
 	GetNonce(addr types.Address) (uint64, bool)
 
+	// GetCapacity returs the current and max capacity of the pool
```
Comment (zivkovicmilos) [inline jsonrpc/blockchain.go]: Typo here `returs`

File: txpool/txpool.go [hunk 2]
```diff
@@ -265,6 +265,11 @@ func (t *TxPool) GetNonce(addr types.Address) (uint64, bool) {
 	return highestNonce + 1, true
 }
 
+// GetCapacity returns the current state of the gauge
```
Comment (zivkovicmilos) [inline txpool/txpool.go]: Please document the exact data that this method returns

File: jsonrpc/filter_manager_test.go [hunk 3]
```diff
@@ -247,6 +247,10 @@ func (m *mockStore) SetAccount(addr types.Address, account *state.Account) {
 	m.accounts[addr] = account
 }
 
+func (m *mockStore) GetCapacity() (uint64, uint64) {
+	return 0, 0
```
Comment (zivkovicmilos) [inline jsonrpc/filter_manager_test.go:251]: Is there any way to have this data be updated as the mockStore is being updated?

**Review by zivkovicmilos — APPROVED** [review]
Looks great 💯 

Left some minor comments that you should check out 👍

File: jsonrpc/filter_manager_test.go (hunk 3 above)
Comment (lazartravica) [reply jsonrpc/filter_manager_test.go:251]: What do you think that would accomplish? @zivkovicmilos

**Review by brkomir — APPROVED** [review]
Looks good!

File: txpool/txpool.go (hunk 2 above)
Comment (lazartravica) [reply txpool/txpool.go]: +1

**Review by lazartravica — APPROVED** [review]
Please add changes from Milos's comments and we're good to go.

File: txpool/txpool.go (hunk 2 above)
Comment (dbrajovic) [reply txpool/txpool.go]: [Changed](https://github.com/0xPolygon/polygon-sdk/pull/262/commits/0f68e9da1b67f49eed2125d022dad4b4c7953b75)

File: jsonrpc/blockchain.go (hunk 1 above)
Comment (dbrajovic) [reply jsonrpc/blockchain.go]: [Fixed](https://github.com/0xPolygon/polygon-sdk/pull/262/commits/0f68e9da1b67f49eed2125d022dad4b4c7953b75)

**Review by Kourin1996 — APPROVED** [review]

File: jsonrpc/filter_manager_test.go (hunk 3 above)
Comment (zivkovicmilos) [reply jsonrpc/filter_manager_test.go:251]: Was just curious for future tests 🙂 

Resolving this since it's not critical 👍

**Pull request merged**